<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Set Graph</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Motor 1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="m1_input_show">
        <property name="text">
         <string>Raw Input</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="m1_input_color">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="m1_input_linetype">
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashdot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m1_setpoint_show">
        <property name="text">
         <string>Setpoint</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="m1_setpoint_color">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="m1_setpoint_linetype">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashdot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="m1_output_show">
        <property name="text">
         <string>Output</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="m1_output_color">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="m1_output_linetype">
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashdot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Motor 2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="m2_input_show">
        <property name="text">
         <string>Raw Input</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="m2_input_color">
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="m2_input_linetype">
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashdot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m2_setpoint_show">
        <property name="text">
         <string>Setpoint</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="m2_setpoint_color">
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="m2_setpoint_linetype">
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashdot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="m2_output_show">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="m2_output_color">
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="m2_output_linetype">
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashdot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
